src/sdk/conference/client.js

```diff
@@ -422,7 +441,16 @@ export const ConferenceClient = function(config, signalingImpl) {
     if (!(stream instanceof StreamModule.RemoteStream)) {
       return Promise.reject(new ConferenceError('Invalid stream.'));
     }
-    const channel = createPeerConnectionChannel();
+    if (stream.source.data) {
```

Comment:
I think "data" is not one kind of source. We should update streaminfo strcuture to identify av/data with "kind" or "media_type" instead.

Reply:
It's better to be `MediaStream` if it carries audio/video data, but it already defined.